hedera-node/hedera-smart-contract-service-impl/src/main/java/module-info.java

```diff
@@ -14,6 +14,8 @@
     requires com.swirlds.config;
     requires com.hedera.node.app.service.mono;
     requires com.hedera.node.app.service.token;
+    requires com.hedera.node.app.service.token.impl;
```

Comment:
Is this needed? Or just `com.hedera.node.app.service.token`

Reply:
Maybe because of `CryptoCreateRecordBuilder`?  I had the same issue when trying to call the util service and trying to get the correct record builder.  Should those record interfaces be moved to the service?

Reply:
yeah probably, did not realize this. I will move them to `com.hedera.node.app.service.token`.

Reply:
I also needed `StakingValidator`.

Reply:
Eliminated this dependency by giving `TokenServiceApi` a `assertValidStakingElection()` method.